CHANGELOG.md

```diff
@@ -1,15 +1,37 @@
 # App Center SDK for React Native Change Log
 
-## Version 2.3.0 (Under active development)
+## Version 2.3.0
 
 ### App Center
 
 * **[Feature]** Catch low memory warning and provide the API to check if it has happened in last session:  `Crashes.hasReceivedMemoryWarningInLastSession()`.
 
 ### App Center Auth
 
+* **[Feature]** App Center Auth logging now includes MSAL logs.
 * **[Fix]** Fix silently signed out on application restart if appcenter-auth module is initialized after other appcenter modules.
 
+#### Android
+
+* **[Fix]** Redirect URIs are now hidden in logs.
+
+### App Center Crashes
+
+#### Android
+
+* **[Feature]** Catch "low memory warning" and provide the API to check if it has happened in last session: Crashes.hasReceivedMemoryWarningInLastSession().
+
+#### iOS
+
+* **[Feature]** Catch "low memory warning" and provide the API to check if it has happened in last session: MSCrashes.hasReceivedMemoryWarningInLastSession().
```

Comment:
It was already mentioned

Reply:
Those are for different platforms. I think it can be simplified a bit (with no Android-iOS breakdown)